Source/Model/SPTooltip.h

```diff
@@ -43,6 +44,10 @@
 	NSPoint			mousePositionWhenOpened;
 	
 	NSString* 		SPTooltipPreferencesIdentifier;
+	
+	BOOL 			gotHeight;
```

Comment:
Can we start using `@property (nonatomic, assign) BOOL gotHeight;` inside `m` file so we have the atomics and assign already in for ARC? :)

Reply:
will do